ci/terraform/oidc/processing-identity.tf

```diff
@@ -1,3 +1,32 @@
+locals {
+  base_policies = [
+    aws_iam_policy.audit_signing_key_lambda_kms_signing_policy.arn,
+    aws_iam_policy.dynamo_user_read_access_policy.arn,
+    aws_iam_policy.dynamo_client_registry_read_access_policy.arn,
+    aws_iam_policy.dynamo_identity_credentials_read_access_policy.arn,
+    aws_iam_policy.lambda_sns_policy.arn,
+    aws_iam_policy.pepper_parameter_policy.arn,
+    aws_iam_policy.redis_parameter_policy.arn,
+    module.oidc_txma_audit.access_policy_arn,
+    local.account_modifiers_encryption_policy_arn,
+    local.client_registry_encryption_policy_arn,
+    local.identity_credentials_encryption_policy_arn,
+    local.user_credentials_encryption_policy_arn
+  ]
+}
+
+module "ipv_processing_identity_role_with_orch_session_access" {
+  source      = "../modules/lambda-role"
+  environment = var.environment
+  role_name   = "ipv-processing-identity-role-with-orch-session-access"
+  vpc_arn     = local.authentication_vpc_arn
+
+  policies_to_attach = var.is_orch_stubbed ? local.base_policies : concat(local.base_policies, [
+    aws_iam_policy.dynamo_orch_session_encryption_key_policy.arn,
+    aws_iam_policy.dynamo_orch_session_logout_policy.arn,
+  ])
+}
+
```

Comment:
This is nice, but will cause us troubles in the future if we have to add more policies. Can we just create a role that has all of the normal policies, and the two orch session ones, and use is_orch_stubbed as a flag on whether to use the old one or the new one?

Reply:
ah good point